UI/js-src/lsmb/MainContentPane.js

```diff
@@ -124,6 +126,7 @@ define([
             domStyle.set(this.domNode, "visibility", "visible");
             domStyle.set(this.domNode, "opacity", "1");
             domClass.replace(this.domNode, "done-parsing", "parsing");
+            connect.publish("lsmb/page-fresh-content", [{}]);
```

Comment:
From https://dojotoolkit.org/reference-guide/1.10/dojo/connect.html:

> As of Dojo 1.7, the preferred way of handling events is to use the new lightweight dojo/on module.

Any reason to use `connect`?

Reply:
Old topic syntax. Switched to topic instead of connect.